src/app/core/store/organizations/organizations.action.ts

```diff
@@ -42,7 +45,15 @@ export enum OrganizationsActionType {
   DELETE_SUCCESS = '[Organizations] Delete :: Success',
   DELETE_FAILURE = '[Organizations] Delete :: Failure',
 
-  SELECT = '[Organizations] Select'
+  SELECT = '[Organizations] Select',
+
+  GET_PERMISSIONS = '[Organizations] Get Permissions',
+  GET_PERMISSIONS_SUCCESS = '[Organizations] Get Permissions :: Success',
+  GET_PERMISSIONS_FAILURE = '[Organizations] Get Permissions :: Failure',
```

Comment:
Simple because of this https://github.com/Lumeer/engine/blob/devel/lumeer-core/src/main/java/io/lumeer/core/facade/OrganizationFacade.java#L99

Same implies for projects, collections and views..

Reply:
So why not changing back-end so it will not only return current user permissions but also other users' permissions in case the user is allowed to see them?

Reply:
That's very good idea. I will do it this way..